infrastructure/terraform/modules/templates-api/variables.tf

```diff
@@ -61,3 +61,20 @@ variable "cognito_config" {
   description = "Cognito config"
 }
 
+variable "template_api_entrypoints" {
```

Comment:
What is the use-case for this being a variable?

Reply:
I couldn't think of a better way to put all the entrypoints I need in one place that is also be used in of the modules. 

Please do suggest a better way.

Reply:
Just don't know the use-case why they would need to be exposed as parameters. Could use locals

```tf
locals {
  template_api_entrypoints = {
    create_template = "src/templates/api/create.ts"
    get_template = "src/templates/api/get.ts"
  }
}
```

Then reference them like `local.template_api_entrypoints.create_template`

Reply:
Updated. Thanks for the suggestion.